src/store/user/actions.js

```diff
@@ -0,0 +1,40 @@
+import firebase from 'firebase'
+import { firebaseAuth } from '../../boot/firebase'
+
+function signUp( { dispatch },payload) {
+    console.log(payload)
+    firebaseAuth.createUserWithEmailAndPassword(payload.email,payload.password).then(data =>{
+      data.user.updateProfile({
+          displayName:this.firstname+this.lastname
```

Comment:
@pavankumarhb all the variables should be always in camelcase, firstname should be changed to firstName and similarly for all the variables